recipes/httpstan/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "httpstan" %}
+{% set version = "4.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stan-dev/httpstan/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f4d81e2ecbf17fce6a03fea7200d10ab03fbad1a7f3db1453464ea12c4fa740c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python  >=3.7
```

Comment:
This is not the correct way to skip python 3.6 for a non-noarch package. see conda-build docs.